lib/statsd/instrument/helpers.rb

```diff
@@ -3,38 +3,9 @@
 module StatsD::Instrument::Helpers
   def capture_statsd_datagrams(client: nil, &block)
     client ||= StatsD.singleton_client
```

Comment:
`capture_statsd_datagrams(client: StatsD.singleton_client, &block)` would make more sense IMO.

Reply:
That doesn't work, unless we propagate this default value out to all the methods that call this. Right now, if I do that tests will fail because we call `capture` on `nil`.